src/core/Akka/Event/LoggingBus.cs

```diff
@@ -214,9 +214,9 @@ private string CreateLoggerName(Type actorClass)
         }
 
         /// <summary>
-        /// Starts the <see cref="StandardOutLogger"/> logger.
+        /// Starts the <see cref="MinimalLogger"/> logger.
         /// </summary>
-        /// <param name="config">The configuration used to configure the <see cref="StandardOutLogger"/>.</param>
+        /// <param name="config">The configuration used to configure the <see cref="MinimalLogger"/>.</param>
         public void StartStdoutLogger(Settings config)
```

Comment:
If I get it right, it's a bit misleading, that if you set your own _MinimalLogger_ then you need to configure it's logging level through `akka.stdout-loglevel` not for example `akka.minimallogger-loglevel`.

Reply:
StandardOutLogger derives from MinimalLogger, it is the barest minimum logger used to log how the system behave without actual loggers running, ie. a logger to log the loggers themselves, it runs at the very start and the very end of the ActorSystem life cycle. The only thing you can make by inheriting MinimalLogger is a replacement for the StandardOutLogger, thus it will use the same settings as StandardOutLogger.